releases/latest/full/Dockerfile.ubi.adoptopenjdk11

```diff
@@ -1,7 +1,7 @@
 FROM adoptopenjdk/openjdk11-openj9:ubi
-ARG LIBERTY_VERSION=20.0.0.12
-ARG LIBERTY_SHA=18deec3a143a6a2ac46c717722a90678c524fa4e
-ARG LIBERTY_BUILD_LABEL=cl201220201111-0736
+ARG LIBERTY_VERSION=21.0.0.1
+ARG LIBERTY_SHA=da39a3ee5e6b4b0d3255bfef95601890afd80709
```

Comment:
This is not same as the one from https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/21.0.0.1/openliberty-runtime-21.0.0.1.zip.sha1